api-spec/definitions/STAC-standalone.yaml

```diff
@@ -1,355 +1,543 @@
-        links:
-          type: object
-          additionalProperties:
-            type: object
-            required:
-              - href
-              - rel
-            properties:
-              href:
-                type: string
-              rel:
-                type: string
-        assets:
-          type: object
-          additionalProperties:
-            type: object
-            required:
-              - href
-            properties:
-              href:
-                type: string
-              name:
-                type: string
-      example:
-        type: Feature
-        id: CS3-20160503_132130_04
-        bbox:
-          - -122.59750209
-          - 37.48803556
-          - -122.2880486
-          - 37.613537207
-        geometry:
-          type: Polygon
-          coordinates:
-            - - - -122.308150179
-                - 37.488035566
-              - - -122.597502109
-                - 37.538869539
-              - - -122.576687533
-                - 37.613537207
-              - - -122.2880486
-                - 37.562818007
-              - - -122.308150179
-                - 37.488035566
-        properties:
-          datetime: '2016-05-03T13:21:30.040Z'
-          provider: 'http://www.cool-sat.com'
-          license: CC-BY-4.0
-        links:
-          self:
-            rel: self
-            href: >-
-              http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04.json
-        assets:
-          analytic:
-            name: 4-Band Analytic
-            href: >-
-              http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04/analytic.tif
-          thumbnail:
-            name: thumbnail
-            href: >-
-              http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04/thumb.png
-    itemProperties:
-      type: object
-      required:
-        - datetime
-      description: provides the core metatdata fields plus extensions
-      properties:
-        datetime:
-          $ref: '#/components/schemas/time'
-        provider:
-          type: string
-          description: Information about the provider
-        license:
-          type: string
-          description: Data license name based on SPDX License List
-      additionalProperties:
-        type: object
-    itemCollectionLinks:
-      type: object
-      properties:
-        next:
-          type: string
-          format: url
-          description: >-
-            A URL to obtain the next paginated data set. If not present, the
-            client should utilize the `nextPageToken`.
-          example: http://api.cool-sat.com/query/gasd312fsaeg/ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk
-tags:
-  - name: STAC
-    description: 'Extension to WFS3 Core to support STAC metadata model and search API'
\ No newline at end of file
+openapi: 3.0.1
+info:
+  title: The SpatioTemporal Asset Catalog API (standalone)
+  version: 0.5.1
```

Comment:
Outdated.

@cholmes How do we make sure we always update all the version numbers hidden in lots of files?